<launch>

  <arg name="rviz" default="true" />
  <param name="use_sim_time" value="false" />

  <node pkg="location" type="locationV2" name="locationV2"  output="screen">
    <param name="mapPCDFileName" value="$(find location)/map/mymap.pcd"/>
    <param name="lidarName" value="/airsim_node/Car1/lidar/LidarSensor1"/>
    <param name="enableSwitchLidarData" value="false"/>
    <param name="switchBufferSize" value="4"/>
    <param name="enablePublishGlobalDenseMap" value="false"/>
  </node>

   <node pkg="tf" type="static_transform_publisher" name="link2_broadcaster" args="0 0 2.4 0 0 0 base_link LidarSensor1 50" />
 

  <node name="MapServer" pkg="map_server" type="map_server" output="screen" args="$(find location)/map/mymap.yaml"/>

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find location)/rviz_cfg/ndt_matching.rviz" />
  </group>


</launch>
